torchdynamo/eval_frame.py

```diff
@@ -25,6 +33,12 @@ def nothing():
     pass
 
 
+def same(left, right):
+    return len(left) == len(right) and all(
```

Comment:
We should be able to merge the `len(left) == len(right)` into the other `same()` function.

Reply:
I copy the same() function from testing.py. I could not directly import it because of cyclic import issue. Is it ok to duplicate the same() function in eval_frame.py or should I move the testing.same() to another file?

Reply:
Move same() from testing.py to utils.py